app/styles/main.scss

```diff
@@ -23,6 +23,10 @@ html, body {
   height: 100%;
 }
 
+.body-container {
+  min-height: calc(100vh - 317px);
```

Comment:
We could even put the nav and footer heights into a variable to make the "magic number" more clear.